docs/content/src/learn/guides/saas-clients/sfdc46/import-contacts-into-salesforce-using-ftp/README.md

```diff
@@ -37,90 +40,95 @@ A Ballerina project needs to be created for the integration use case explained a
 below to create the project and modules. You can learn about the Ballerina project and modules in this 
 [guide](https://ei.docs.wso2.com/en/latest/ballerina-integrator/develop/using-modules/#creating-a-project).
 
-1. Create a new project.
+#### 1. Create a new project.
```

Comment:
This format is a bit different from what others have done. It's good if we can be consistent in all guides regarding the structure and formatting. Is there a reason for changing this?

Reply:
This changed since ordering is not showing properly on the website.